TESTS/simple/connect/main.cpp

```diff
@@ -39,45 +31,39 @@ void smcc_register(void) {
 
     iteration = atoi(_value);
 
-    // Storage definition.
-#if defined(MBED_CONF_APP_TEST_BLOCK_DEVICE_OBJECT)
-    MBED_CONF_APP_TEST_BLOCK_DEVICE_OBJECT
-#else
-    SDBlockDevice bd(MBED_CONF_APP_SPI_MOSI, MBED_CONF_APP_SPI_MISO,
-            MBED_CONF_APP_SPI_CLK, MBED_CONF_APP_SPI_CS);
-#endif
-    FATFileSystem fs("sd", &bd);
-
     // Connection definition.
-#if defined(MBED_CONF_APP_TEST_SOCKET_OBJECT)
-    MBED_CONF_APP_TEST_SOCKET_OBJECT
-#else
-    EthernetInterface net;
-#endif
-
-#if defined(MBED_CONF_APP_TEST_SOCKET_CONNECT)
-    nsapi_error_t status = MBED_CONF_APP_TEST_SOCKET_CONNECT
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE == ETHERNET
+    NetworkInterface *net = NetworkInterface::get_default_instance();
```

Comment:
So we stopped supporting 6LoWPAN etc. now? I don't like having this in the library, I want it in the application.